test/extensions/bootstrap/wasm/wasm_test.cc

```diff
@@ -112,14 +112,15 @@ class WasmTestMatrix : public WasmTestBase,
   void createWasm() { WasmTestBase::createWasm(std::get<0>(GetParam())); }
 };
 
+// TODO(shikugawa): Add tests for Rust logging with proxy-wasm-rust-sdk.
 INSTANTIATE_TEST_SUITE_P(RuntimesAndLanguages, WasmTestMatrix,
                          testing::Combine(testing::Values("v8"
 #if defined(ENVOY_WASM_WAVM)
                                                           ,
                                                           "wavm"
 #endif
                                                           ),
-                                          testing::Values("cpp", "rust")));
+                                          testing::Values("cpp")));
```

Comment:
If this PR disables `wasm_vm_test` and removes `rust` from `wasm_test`, then there are no Rust tests executed at all...

Reply:
`wasm_test` is temporary disabled for my debugging of OOM killed in CI.